wiki/Tournaments/COT/2022/en.md

```diff
@@ -0,0 +1,88 @@
+| Grand Finals | 2022-03-19/2022-03-20 |
+
+## Prizes
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 18 months of osu!supporter or 3000₽, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 12 months of osu!supporter or 2000₽ |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 4 month of osu!supporter or 1000₽ |
+
+## Organisation
+
+COT 2022 was run by various osu! community members mostly hailing from Commonwealth of Independent States.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_UA] [bernkastel feet](https://osu.ppy.sh/users/13681464), ![][flag_RU] [ThankYou](https://osu.ppy.sh/users/4571241) |
+| Mappool selector | ![][flag_UA] [bernkastel feet](https://osu.ppy.sh/users/13681464), ![][flag_RU] [Xetopia](https://osu.ppy.sh/users/6689101), ![][flag_RU] [Rebbel](https://osu.ppy.sh/users/4595355), ![][flag_RU] [ThankYou](https://osu.ppy.sh/users/4571241), ![][flag_RU] [_Insomnia](https://osu.ppy.sh/users/8928855), ![][flag_CL] [Pekorrat](https://osu.ppy.sh/users/1250096) |
+| Streamer | ![][flag_RU] [ThankYou](https://osu.ppy.sh/users/4571241), ![][flag_UA] [bernkastel feet](https://osu.ppy.sh/users/13681464), ![][flag_RU] [2zz](https://osu.ppy.sh/users/8201267) |
+| Commentator | ![][flag_RU] [Nelly](https://osu.ppy.sh/users/4741164),  ![][flag_RU] [_Insomnia](https://osu.ppy.sh/users/8928855), ![][flag_UA] [bernkastel feet](https://osu.ppy.sh/users/13681464), ![][flag_RU] [ThankYou](https://osu.ppy.sh/users/4571241) |
+| Referee | ![][flag_RU] [_Insomnia](https://osu.ppy.sh/users/8928855), ![][flag_UA] [bernkastel feet](https://osu.ppy.sh/users/13681464), ![][flag_RU] [2zz](https://osu.ppy.sh/users/8201267) |
+| Designers | ![][flag_RU] [ThankYou](https://osu.ppy.sh/users/4571241), ![][flag_RU] [2zz](https://osu.ppy.sh/users/8201267) |
+| Spreadsheet manager | ![][flag_UA] [bernkastel feet](https://osu.ppy.sh/users/13681464) |
+| Wiki editor | ![][flag_RU] [Nelly](https://osu.ppy.sh/users/4741164) |
+
+## Links
+
+- **[Main spreadsheet](https://docs.google.com/spreadsheets/d/1iie_zXj1GbS_Sx4MyWdGOYD9raofZ46uomMbVW6wm48/edit?usp=sharing)**
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1493650)
+- [Discord server](https://discord.gg/ZqCdWBKGrM)
+- [Challonge brackets](https://challonge.com/ru/cot22)
+- [Livestream channel](https://www.twitch.tv/playosuru/)
+
+## Ruleset
+
+### General rules
+
+1. Map scoring is based on **[ScoreV2](/wiki/Score#scorev2).**
+2. The match lobby will be created by the referee, and the players will be invited.
+3. Players have a strict 10-minute grace period available to play the match. If 10 minutes pass and a player does not show up the match, the other player will receive a win by default.
+4. Warm ups aren't allowed.
+5. During the lobby it is not allowed to show any form of harassment during livestreaming. By organizers after breaking the rule the player may be disqualified.
```

Comment:
```suggestion
5. During the lobby it is not allowed to show any form of harassment during livestreaming. As per the staff decision, after breaking the rule the player may be disqualified.
```

Reply:
rule is kinda sus for only disallowing harassment when livestreamed but ok

```suggestion
5. It is not allowed to show any form of harassment in a livestreamed lobby. As per staff decision, a player may be disqualified for breaking this rule.
```